.appveyor.yml

```diff
@@ -41,6 +41,11 @@ test_script:
   # FIXME: Exclude Appveyor from running `lantern` smoketest until we fix the flake.
   - yarn smoke -j=1 --retries=2 a11y errors oopif pwa pwa2 pwa3 dbw redirects seo offline byte perf metrics
 
+on_success:
+  # Remove module resolution symlinks; this avoids 'Unable to save cache'
+  # errors when appveyor runs 7z.exe to save the build cache
+  - yarn unlink
```

Comment:
this covers undoing both `yarn link` and `yarn link lighthouse` right? (though now I'm not really sure why we do both wouldn't they do the same thing?)

Reply:
Strictly speaking it's safest to run `yarn unlink lighthouse` first.

- `yarn link` creates a global reference to the local lighthouse working directory
- `yarn link lighthouse` creates a local link from `node_modules/lighthouse` to the global reference
- `yarn unlink lighthouse` removes the local reference from `node_modules`
- `yarn unlink` removes the global reference

I'll push a small change to include the `yarn unlink lighthouse` step.